docs/subscriptions.rst

```diff
@@ -0,0 +1,69 @@
+Subscriptions
+#############
+
+What is a Subscription?
+***********************
+
+A `Globus subscription`_ is a paid, organization-level plan that adds additional
+capabilities across the entire `Globus`_ platform.  It is not a personal account; the
+subscription is purchased by an institution, research group, or company and applied to
+users in that organization.
+
+While an `organizational subscription is an add-on`_, many Globus features are
```

Comment:
I might make the link to `organizational subscription` instead of including the words 'is an add-on', especially as the linked page doesn't emphasize 'many features are included'.